### PR TITLE
Fix for Windows with graphviz installed from conda

### LINKED ISSUE #25: Fix windows-bat.patch
Please, return [the master version of ``windows-bat.patch``](https://github.com/conda-forge/python-graphviz-feedstock/blob/master/recipe/windows-bat.patch) (it's empty) to  [the previous one](https://github.com/conda-forge/python-graphviz-feedstock/blob/d83128c4d98ac54ef346b2b529a7cb0dd59e66b4/recipe/windows-bat.patch).

Without this patch the python-graphviz doesn't work correctly under Windows with ``graphviz`` from conda.

See also xflr6/graphviz/issues/65

### PR DESCRIPTION
**Pull request by StrikerRUS**
This reverts commit 2a4f1a8145bbf79823c40d723821eb16c9689235.

Should fix #25.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jakirkham** [comment]
This is great! Thanks @StrikerRUS. 😀

Could you please also [add this patch to `source` in `recipe/meta.yaml`]( https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#patches )?

**Comment by StrikerRUS** [comment]
@jakirkham Done!

I know less than nothing about recipe building, but it seems to me that this is not needed anymore: 6a6e7ebada33f8e8c843a73eadfecdb1133ec969.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,6 +9,8 @@ source:
   fn: python-graphviz-{{ version }}.zip
   url: https://pypi.io/packages/source/g/graphviz/graphviz-{{ version }}.zip
   sha256: {{ sha256 }}
+  patches:
+    - windows-bat.patch
 
 build:
   number: 0
```
Comment (scopatz) [inline recipe/meta.yaml]: Can you bump the build number to 1 please?

**Comment by jakirkham** [comment]
Ah good point @StrikerRUS! Forgot we have to do something special to apply the patch. Please feel free to revert that last commit.

File: recipe/meta.yaml (hunk 1 above)
Comment (StrikerRUS) [reply recipe/meta.yaml]: @scopatz Sure, done!

**Comment by StrikerRUS** [comment]
@jakirkham OK, reverted.

**Comment by scopatz** [comment]
It couldn't apply the patch:

```
INFO:conda_build.source:Success
Extracting download
source tree in: /home/conda/feedstock_root/build_artifacts/python-graphviz_1559747118119/work
export PREFIX=/home/conda/feedstock_root/build_artifacts/python-graphviz_1559747118119/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place
export BUILD_PREFIX=/home/conda/feedstock_root/build_artifacts/python-graphviz_1559747118119/_build_env
export SRC_DIR=/home/conda/feedstock_root/build_artifacts/python-graphviz_1559747118119/work
~/recipe_root ~/feedstock_root/build_artifacts/python-graphviz_1559747118119/work
~/feedstock_root/build_artifacts/python-graphviz_1559747118119/work
patching file graphviz/backend.py
Hunk #1 FAILED at 114 (different line endings).
1 out of 1 hunk FAILED -- saving rejects to file graphviz/backend.py.rej
```

**Comment by jakirkham** [comment]
IIRC the line endings should be CRLF, but I could be wrong about this.

**Comment by StrikerRUS** [comment]
Any updates, guys?..

**Comment by jakirkham** [comment]
Did you try my suggestion above?

**Comment by StrikerRUS** [comment]
@jakirkham You mean set line endings to `CRLF` in `windows-bat.patch`?

**Comment by scopatz** [comment]
yeah, I think that is what he means.

**Comment by StrikerRUS** [comment]
Seems that `CRLF` didn't help.

File: recipe/windows-bat.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
+--- graphviz/backend.py
++++ graphviz/backend.py
+@@ -114,6 +114,8 @@
+     suffix = '.'.join(reversed(format_arg))
+     format_arg = ':'.join(format_arg)
+
```
Comment (jakirkham) [inline recipe/windows-bat.patch]: This needs one space before the line break.

```
patch: **** malformed patch at line 6: 

patching file graphviz/backend.py
```

ref: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=41373

**Comment by jakirkham** [comment]
I think that helped, but there is now a different issue. Please see the comment above for details.

**Comment by StrikerRUS** [comment]
@jakirkham 

>This needs one space before the line break.

Done!

**Comment by jakirkham** [comment]
We may need to adjust the patch a little bit. See [this code for context]( https://github.com/xflr6/graphviz/blob/0.11/graphviz/backend.py#L113-L114 ).

**Comment by StrikerRUS** [comment]
@jakirkham 
> We may need to adjust the patch a little bit. See this code for context.

Please check my latest commit.

**Comment by jakirkham** [comment]
Looks great @StrikerRUS! Thanks for all of the work here.

Please let us know if the new packages work correctly.

Also please feel free to add yourself as a maintainer on this feedstock if you're interested. 🙂

**Comment by StrikerRUS** [comment]
@jakirkham
> Please let us know if the new packages work correctly.

 Just checked the newest build: works like a charm!

> Also please feel free to add yourself as a maintainer on this feedstock if you're interested. 🙂

Thanks a lot for the suggestion, but I'd prefer to remain ordinary user because this package is only non-required dependency for the package I maintain. Also, I'm not familiar with conda recipes at all.

@jakirkham @scopatz 
Thanks you guys very much for your patience  and great support during the whole process.

**Comment by scopatz** [comment]
Our pleasure!

**Pull request merged**